library/multiprocessing.po

```diff
@@ -660,6 +899,12 @@ msgid ""
 "meth:`~queue.Queue.join` methods introduced into Python 2.5's :class:`queue."
 "Queue` class."
 msgstr ""
+"Les types :class:`Queue`, :class:`SimpleQueue` et :class:`JoinableQueue` "
+"sont des queues :abbr:`FIFO (first-in, first-out)` multi-producteurs et "
```

Comment:
(premier arrivé, premier sorti , abbr FIFO * first-in, first-out* en anglais).

Reply:
Je ne sais pas précisément comment le formuler/formater, l'italique ne semblant pas être possible dans le texte d'une abréviation.